### PR TITLE
Assignment-2 | Component | React-Hooks(useState,useContext,useEffect,useReducer) | Custom Hooks

### PR DESCRIPTION
**Pull request by kaushalyad321**

### CODE REVIEW

**Review by sumitbaudh22 — COMMENTED** [review]
Task 3 not working properly

**Review by sumitbaudh22 — COMMENTED** [review]
Task 19 not working properly

File: Assignment-2/app/src/Component/DisplayTaskList.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Task7 from '../Component/taskComponents/Task7';
+import Task8 from '../Component/taskComponents/Task8';
+
+
+const DisplayTaskList = () => {
+  return (
+    <div>
+      <div style={{fontSize:'60px',display:'flex',justifyContent:'center'}}>TASK</div>
+      <ol>
+       <Task1/>
```
Comment (sumitbaudh22) [inline Assignment-2/app/src/Component/DisplayTaskList.js]: I think we can use map here instead of creating tasks separately

Comment (kaushalyad321) [reply Assignment-2/app/src/Component/DisplayTaskList.js]: ok but need to store in the array all the separate task then we can render the task using map

Comment (sumitbaudh22) [reply Assignment-2/app/src/Component/DisplayTaskList.js]: No, I don't think so. All the task component are same just they have different task string that can be passed as a prop.
Please connect with me for more clarity

**Comment by kaushalyad321** [comment]
> Task 3 not working properly



> Task 3 not working properly
now solved the issue and stored the task in object with id and task will be deleted with specific id


> Task 19 not working properly

updated plz check and let me know

File: Assignment-2/app/src/Component/DisplayTaskList.js (hunk 1 above)
Comment (kaushalyad321) [reply Assignment-2/app/src/Component/DisplayTaskList.js]: > I think we can use map here instead of creating tasks separately
also worked on this

**Review by sumitbaudh22 — APPROVED** [review]

**Pull request merged**